src/containers/pages/FocusInvestigation/active/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -4945,18 +4890,7 @@ exports[`containers/pages/ActiveFocusInvestigation renders ActiveFocusInvestigat
         "calls": Array [
           Array [
             0,
-            Object {
-              "adhoc_filters": Array [
-                Object {
-                  "clause": "WHERE",
-                  "comparator": "FI",
-                  "expressionType": "SIMPLE",
-                  "operator": "==",
-                  "subject": "plan_intervention_type",
-                },
-              ],
-              "row_limit": 2000,
-            },
+            Object {},
```

Comment:
I think that this change should not happen.  Have we stopped calling the Superset service with this filter?